src/services/sync/strategies/CSVSyncer.js

```diff
@@ -200,13 +200,18 @@ class CSVSyncer extends mix(Syncer).with(ClassImporter) {
 	}
 
 	static sanitizeRecords(records) {
+
 		const requiredAttributes = ATTRIBUTES.filter((a) => a.required).map((a) => a.name);
 		const mappingFunction = buildMappingFunction(records[0]);
 		const processed = [];
 		records.forEach((record) => {
+
 			const mappedRecord = mappingFunction(record);
 			if (requiredAttributes.every((attr) => !!mappedRecord[attr])) {
 				mappedRecord.email = mappedRecord.email.trim().toLowerCase();
+				if (!this.dateValidator(mappedRecord.birthday)) { // todo: put validation elsewhere
```

Comment:
I would argue the location is nearly perfect, but there needs to be some kind of error/warning logging if a value is missing or incorrect. Missing values shouldn't be a problem, as this attribute is optional (and it already works without birthday), but if there's something wrong, it should appear in the stats